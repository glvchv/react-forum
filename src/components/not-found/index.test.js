import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '.';
import { BrowserRouter } from 'react-router-dom';

jest.mock('../header', () => 'Header');
jest.mock('../footer', () => 'Footer');

describe('Not-found component', () => {
    it('should render component', () => {
        const tree = renderer.create(
            <BrowserRouter>
                <NotFound />
            </BrowserRouter>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})