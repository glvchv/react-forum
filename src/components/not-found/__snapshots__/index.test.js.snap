// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not-found component should render component 1`] = `
Array [
  <Header />,
  <div
    className="container"
  >
    <h1
      className="status"
    >
      404
    </h1>
    <h2
      className="msg"
    >
      Oops... Seems like this page does not exist!
    </h2>
    <a
      href="/"
      onClick={[Function]}
    >
      <button
        className="gbBtn"
      >
        <b>
          Home
        </b>
      </button>
    </a>
  </div>,
  <Footer />,
]
`;
